### PR TITLE
Add my agora instance

### PR DESCRIPTION
**Pull request by yzernik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Public Agora instances:
 
 - [agora.download](https://agora.download): Development instance on mainnet
 - [test.agora.download](https://test.agora.download): Development instances on testnet
+- [tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion](http://tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion): Instance belonging to [@yzernik](https://github.com/yzernik) (accessible over Tor)
```
Comment (casey) [inline README.md:18]:
```suggestion
- [tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion](http://tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion): [@yzernik's](https://github.com/yzernik) instance, accessible over Tor
```

**Comment by casey** [comment]
Nice, merging! How was getting it running over Tor?

**Comment by yzernik** [comment]
Umbrel automatically sets up the Tor hidden service, so it's very easy.

```
cat umbrel/tor/data/app-agora/hostname
```

**Comment by casey** [comment]
Dope, that's sick.

**Comment by casey** [comment]
Thanks for rebasing! Merged.

**Pull request merged**